src/__tests__/ClickActionFocus.test.tsx

```diff
@@ -64,5 +66,56 @@ test("should not select on enter/space keydown when click action is focus", () =
     );
   }
   fireEvent.keyDown(node, { key: " " });
-  expect(nodes[0]).not.toHaveAttribute("aria-selected");
+  expect(nodes[0]).toHaveAttribute("aria-selected", "false");
+});
+
+test("should not select on control keydown when click action is focus", () => {
+  const { queryAllByRole } = render(<FocusClickActionTreeView />);
+
+  const nodes = queryAllByRole("treeitem");
+  nodes[0].focus();
+
+  if (document.activeElement == null) {
+    throw new Error(
+      `Expected to find an active element on the document (after focusing the first element with role["treeitem"]), but did not.`
+    );
+  }
+
+  fireEvent.keyDown(document.activeElement, { key: "a", ctrlKey: true });
+  nodes.forEach((x) => expect(x).toHaveAttribute("aria-selected", "false"));
+
+  fireEvent.keyDown(document.activeElement, {
+    key: "Home",
+    ctrlKey: true,
+    shiftKey: true,
+  });
+  nodes.forEach((x) => expect(x).toHaveAttribute("aria-selected", "false"));
+
+  fireEvent.keyDown(document.activeElement, {
+    key: "End",
+    ctrlKey: true,
+    shiftKey: true,
+  });
+  nodes.forEach((x) => expect(x).toHaveAttribute("aria-selected", "false"));
+});
+
+test("should not select on shift + arrow up/down keydown when click action is focus", () => {
+  const { queryAllByRole } = render(<FocusClickActionTreeView />);
+
+  const nodes = queryAllByRole("treeitem");
+  nodes[0].focus();
+
+  if (document.activeElement == null) {
+    throw new Error(
+      `Expected to find an active element on the document (after focusing the first element with role["treeitem"]), but did not.`
+    );
+  }
+  fireEvent.keyDown(document.activeElement, {
+    key: "ArrowDown",
+    shiftKey: true,
+  });
+  nodes.forEach((x) => expect(x).toHaveAttribute("aria-selected", "false"));
+  nodes[4].focus();
```

Comment:
Nit: maybe add some comment why we use exactly the node with index `4`

Reply:
Just want to ensure that the focus node has an item to shift up to.